src/Filters/coefficients.jl

```diff
@@ -380,3 +380,4 @@ SecondOrderSections(f::FilterCoefficients) = SecondOrderSections(ZeroPoleGain(f)
 *(f1::Biquad, fs::Biquad...) = SecondOrderSections([f1, fs...], 1)
 *(f1::SecondOrderSections, f2::Biquad) = SecondOrderSections([f1.biquads; f2], f1.g)
 *(f1::Biquad, f2::SecondOrderSections) = SecondOrderSections([f1; f2.biquads], f2.g)
+^(f::FilterCoefficients, n::Integer) = *(fill(f, n)...)
```

Comment:
What happens if `n <= 0`?

Reply:
Good point.  I suppose for negative values of n it could return the reciprocal of the positive n value?  Values of n of 0 or 1 also need to be handled.